components/identifier-generator/front/src/feature/components/NomenclatureEdit.tsx

```diff
@@ -0,0 +1,252 @@
+import React, {FC, useCallback, useEffect, useState} from 'react';
+import {
+  AttributesIllustration,
+  Button,
+  Checkbox,
+  Modal,
+  NumberInput,
+  Pagination,
+  Placeholder,
+  Search,
+  SectionTitle,
+  Table,
+  useBooleanState,
+} from 'akeneo-design-system';
+import {Styled} from './Styled';
+import {useGetFamilyNomenclatureValues, useGetNomenclature, useSaveNomenclature} from '../hooks';
+import {OperatorSelector} from './OperatorSelector';
+import {Nomenclature, NomenclatureFilter, NomenclatureValues, Operator} from '../models';
+import {NomenclatureLineEdit} from './NomenclatureLineEdit';
+import {NomenclatureValuesDisplayFilter} from './NomenclatureValuesDisplayFilter';
+import {Violation} from '../validators';
+import {NotificationLevel, useNotify, useTranslate} from '@akeneo-pim-community/shared';
+
+type NomenclatureEditProps = {
+  itemsPerPage?: number;
+};
+
+const NomenclatureEdit: FC<NomenclatureEditProps> = ({itemsPerPage = 25}) => {
+  const translate = useTranslate();
+  const notify = useNotify();
+  const [isOpen, open, close] = useBooleanState();
+  const [nomenclature, setNomenclature] = useState<Nomenclature | undefined>(undefined);
+  const {data: fetchedNomenclature} = useGetNomenclature('family');
+  const [filter, setFilter] = useState<NomenclatureFilter>('all');
+  const [valuesToSave, setValuesToSave] = useState<NomenclatureValues>({});
+  const [violations, setViolations] = useState<Violation[]>([]);
+  const {
+    data: nomenclatureLines,
+    page,
+    setPage,
+    search,
+    setSearch,
+    totalValuesCount,
+    filteredValuesCount,
+    hasValueInvalid,
+  } = useGetFamilyNomenclatureValues(nomenclature, filter, valuesToSave, itemsPerPage);
+  const {save, isLoading: isSaving} = useSaveNomenclature();
+
+  const onFilterChange = (value: NomenclatureFilter) => {
+    if (nomenclature) setNomenclature({...nomenclature, values: valuesToSave});
+    setFilter(value);
+  };
+
+  useEffect(() => {
+    if (fetchedNomenclature) {
+      setNomenclature(fetchedNomenclature);
+      setValuesToSave(fetchedNomenclature.values);
+    }
+  }, [fetchedNomenclature]);
+
+  const handleValueToSaveChange = useCallback(
+    (familyCode, value) => {
+      setValuesToSave({...valuesToSave, [familyCode]: value});
+    },
+    [valuesToSave]
+  );
+
+  const handleChangeOperator = useCallback(
+    operator => {
+      if (nomenclature) {
+        setNomenclature({...nomenclature, operator: operator});
```

Comment:
small change but :heart_eyes: 
```suggestion
        setNomenclature({...nomenclature, operator});
```